app/assets/javascripts/users.coffee

```diff
@@ -1,3 +1,3 @@
-# Place all the behaviors and hooks related to the matching controller here.
-# All this logic will automatically be available in application.js.
-# You can use CoffeeScript in this file: http://coffeescript.org/
+# Place all the behaviors and hooks related to the matching controller here.

```

Comment:
In the future, if we aren't using these, let's remove them as they're unnecessary.

Reply:
Do you mean remove the entire coffeescript file?

Reply:
yup, unless you plan to use it in the future. you can always re-add it as well.